utils/flags/flags.go

```diff
@@ -471,6 +471,12 @@ func GetCliFlags() []cli.Flag {
 			DefaultText: "false, ie disable metrics",
 			EnvVars:     []string{"BAZEL_REMOTE_ENABLE_ENDPOINT_METRICS"},
 		},
+		&cli.BoolFlag{
+			Name:        "http_metrics_prefix",
+			Usage:       "Prefix to apply to exported http metrics",
```

Comment:
This needs to be updated after the refactoring- how about something like "Whether or not to add a 'bazel_remote_' prefix to HTTP metrics" ?

Also, should the prefix end with an underscore?

Reply:
> This needs to be updated after the refactoring- how about something like "Whether or not to add a 'bazel_remote_' prefix to HTTP metrics" ?

Good catch.

> Also, should the prefix end with an underscore?

No, the underscore between the prefix and the underlying metric gets injected automagically.